### PR TITLE
rsweb-7349-cta-refactor-and-width-change

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-7349

style(cta):removing 100% cta width
Removing 100% width as well as refactoring and adjusting design for cta pattern

Changes can be seen/tested on 
http://localhost:9000/derek/assets/buttons
http://localhost:9000/derek/examples/solutions and
http://localhost:9000/derek/examples/homepage

Looking to make sure this won't break buttons in production either. Should try and put on staging for full visual testing.

### CODE REVIEW

File: styleguide/_themes/global/scss/buttons.scss [hunk 1]
```diff
@@ -51,22 +52,6 @@
...
-      background-color: transparent;
-      color: $white;
-
-      &:hover {
-        background-color: $brand-secondary;
-      }
-    }
-  }
-}
-
```
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/buttons.scss]: What does this have to do with CTAs?

**Comment by WidgetsBurritos** [comment]
There's a sass compilation issue as well.  See: 
https://travis-ci.org/rackerlabs/zoolander/jobs/169586850#L330

File: styleguide/_themes/global/scss/buttons.scss (hunk 1 above)
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/buttons.scss]: Okay.

**Comment by ejchiefs** [comment]
👍

**Comment by brantdarilek** [comment]
@amberRrucker 
Just noticed an issue with buttons on the modal form being left justified.

screen shot 2016-11-08 at 4 22 50 pm

**Comment by brantdarilek** [comment]
@amberRrucker 
Looks like there is an issue with the search button as well.
https://github.rackspace.com/rswebteam/www/pull/1389#issuecomment-318968

**Comment by brantdarilek** [comment]
@amberRrucker 
Per conversation with @ejchiefs, could add the following here:
`font-size: 1.2rem;`
https://github.com/rackerlabs/zoolander/blob/master/styleguide/_themes/derek/scss/globalElements/mainnav.scss#L265

**Comment by amberRrucker** [comment]
@brantdarilek I made that change. I have no problems with the buttons in the modal being left aligned since that's how all the other buttons are going to behave.

**Comment by WidgetsBurritos** [comment]
👍  on changes.
